tests/storage_locations/storage_locations.cpp

```diff
@@ -46,7 +46,11 @@ int main(int argc, char *argv[])
     qDebug() << "Application:" << QCoreApplication::applicationName();
 
     qDebug() << "== QStandardPaths ==";
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Comment:
The ::DataLocation is deprecated already in Qt5 and not used here except this debug. Just change it to AppDataLocation without version check.